src/cmd/sign.js

```diff
@@ -107,6 +110,19 @@ export default function sign(
         downloadDir: artifactsDir,
       });
 
+      if (updateLink && updateLink.length > 0 && signingResult.success) {
+        let extensionID;
+        if (signingResult.id) {
+          extensionID = signingResult.id;
+        } else {
+          extensionID = id;
+        }
+        await generateNewUpdateManifest(
+          extensionID, manifestData.version, signingResult.downloadedFiles[0],
```

Comment:
Keep in mind that `signingResult.downloadedFiles` is an array of many files. I'm not entirely sure if newer WebExtensions will have multiple files but historically AMO would sign one file for desktop, one file for Android, and so on (this was mostly for old add-ons with binary components though). In any case, I suppose we should try to support an array of files?